lab4.md

```diff
@@ -158,6 +158,71 @@ In Python code you'd type in `os.getenv("Http_X_Output_Mode")`.
 
 You can see that all other HTTP context is also provided such as `Content-Length` when the `Http_Method` is a `POST`, the `User_Agent`, Cookies and anything else you'd expect to see from a HTTP request.
 
+### Test combined output
+ 
+By default function `stdout/stderr` is combined. You have an option to write `stderr` to the container logs and use `stdout` for function response using the `combine_output` flag.
```

Comment:
Let's briefly define the purpose of stdout and stderr and why we have chosen to combine them by default.